test/e2e/main_test.go

```diff
@@ -123,6 +123,10 @@ func testMain(m *testing.M) int {
 	if err != nil {
 		log.Fatalf("Error when deploying Antrea: %v", err)
 	}
+	// Collect PodCIDRs after Antrea is running as Antrea is responsible for allocating PodCIDRs in some cases.
+	if err = testData.collectPodCIDRs(); err != nil {
```

Comment:
I don't see any polling mechanism in `collectPodCIDRs`. After `ensureAntreaRunning` returns successfully, is it guaranteed that all the Nodes have been updated with a PodCIDR (when using Antrea's NodeIPAM)? No risk of a delay?
If this is true, would you mind explaining why in the comment?